src/ui/annotations.ts

```diff
@@ -1073,43 +1079,348 @@ export class AnnotationSchemaView extends Tab {
+    ];
+
+    const DROPDOWN_OPTIONS = [
+      { header: "General", items: ["float32", "Boolean"] },
+      { header: "Enum", items: ["uint8", "uint16"] },
+      { header: "Colour", items: ["RGB", "RGBa"] },
+      { header: "Integer", items: ["int8", "int16", "int32"] }
+    ];
+
+    const SECTION_ICONS: Record<string, string> = {
+      "Enum": svg_format_size,
+      "Colour": svg_palette,
+      "Integer": svg_numbers
+    };
+
+    const ITEM_ICONS: Record<string, string> = {
+      "float32": svg_numbers,
+      "Boolean": svg_check
+    };
+
+    const createTableCell = (content: string | HTMLElement, className: string = "_"): HTMLDivElement => {
+      const cell = document.createElement("div");
+      cell.classList.add("neuroglancer-annotation-schema-cell", className);
+
+      if (typeof content === "string") {
+        cell.textContent = content;
+      } else {
+        cell.appendChild(content);
+      }
+
+      return cell;
+    };
+
+    const createInputElement = (config: {
+      type: string;
+      value?: string;
+      name: string;
+      id: string;
+      className?: string;
+      placeholder?: string;
+    }): HTMLInputElement => {
+      const input = document.createElement("input");
+      input.type = config.type;
+      input.value = config.value || "";
+      input.name = config.name;
+      input.id = config.id;
+      if (config.className) input.classList.add(config.className);
+      if (config.placeholder) input.placeholder = config.placeholder;
+      return input;
+    };
+
+    const getTypeIcon = (type: string): string => {
+      return ITEM_ICONS[type] ||
+        SECTION_ICONS[
+        Object.keys(SECTION_ICONS).find(section =>
+          DROPDOWN_OPTIONS.find(d => d.header === section)?.items.includes(type)
+        ) || ""
+        ] || svg_format_size;
+    };
+
+    const createDefaultValueInput = (type: string, index: number): HTMLElement => {
+      const container = document.createElement("div");
+      container.style.width = "100%";
+      container.style.display = "flex";
+
+      switch (type) {
+        case "Boolean":
+          container.appendChild(createInputElement({
+            type: "checkbox",
+            name: `boolean-${index}`,
+            id: `boolean-${index}`
+          }));
+          break;
+
+        case "RGB":
+          container.appendChild(createInputElement({
+            type: "color",
+            name: `rgb-${index}`,
+            id: `rgb-${index}`
+          }));
+          break;
+
+        case "RGBa":
+          container.appendChild(createInputElement({
+            type: "color",
+            name: `rgba-${index}`,
+            id: `rgba-${index}`
+          }));
+
+          const alphaInput = createInputElement({
+            type: "number",
+            value: "0.1",
+            name: `alpha-${index}`,
+            id: `alpha-${index}`,
+            className: "schema-default-input"
```

Comment:
I'd be inclined to just stick to the naming of `neuroglancer-schema-default-input` because then if someone happens to use neuroglancer as a library they are extremely unlikely to have any style conflicts, and it keeps the naming consistent